aviary/variable_info/functions.py

```diff
@@ -166,9 +165,9 @@ def setup_trajectory_params(
     are being used in the trajectory, and for the variables which are
     not options it adds them as a parameter of the trajectory.
     """
-    # TODO: variables_to_add might be an unused option.
+    # TODO: variables_to_add is required, so should be an arg, not a kwarg.
```

Comment:
Is it guaranteed to always be required, even in the case of all custom subsystems? Also, can we default it to an empty list and eliminate this code block?

Reply:
It's more that every time we call it right now in Aviary, we give it that argument.